spring-security-oauth2/src/main/java/org/springframework/security/oauth2/provider/expression/OAuth2SecurityExpressionMethods.java

```diff
@@ -91,7 +91,7 @@ public boolean clientHasRole(String role) {
 
 	/**
 	 * Check if the OAuth2 client (not the user) has one of the roles specified. To check the user's roles see
-	 * {@link #clientHasAnyRole(String...)}.
+	 * {@link ExpressionUrlAuthorizationConfigurer#hasAnyRole(String...)}.
```

Comment:
This should be `OAuth2ExpressionUtils#clientHasAnyRole(Authentication, String...)`